dali/test/python/test_dali_cpu_only.py

```diff
@@ -817,4 +818,159 @@ def test_separated_exec_setup():
+    "decoders.image_slice",
+    "decoders.image_random_crop",
+    "decoders.audio",
+    "external_source",
+    "stack",
+    "reductions.variance",
+    "reductions.std_dev",
+    "reductions.rms",
+    "reductions.mean",
+    "reductions.mean_square",
+    "reductions.max",
+    "reductions.min",
+    "reductions.sum",
+    "transforms.translation",
+    "transforms.rotation",
+    "transforms.scale",
+    "transforms.combine",
+    "transforms.shear",
+    "transforms.crop",
+    "transform_translation",
+    "crop",
+    "constant",
+    "dump_image",
+    "numpy_reader",
+    "tfrecord_reader",
+    "file_reader",
+    "sequence_reader",
+    "mxnet_reader",
+    "caffe_reader",
+    "caffe2_reader",
+    "coco_reader",
+    "readers.file",
+    "readers.sequence",
+    "readers.tfrecord",
+    "readers.mxnet",
+    "readers.caffe",
+    "readers.caffe2",
+    "readers.coco",
+    "readers.numpy",
+    "coin_flip",
+    "uniform",
+    "random.uniform",
+    "random.coin_flip",
+    "random.normal",
+    "random_bbox_crop",
+    "python_function",
+    "rotate",
+    "brightness_contrast",
+    "hue",
+    "brightness",
+    "contrast",
+    "hsv",
+    "color_twist",
+    "saturation",
+    "old_color_twist",
+    "shapes",
+    "crop",
+    "color_space_conversion",
+    "cast",
+    "resize",
+    "gaussian_blur",
+    "crop_mirror_normalize",
+    "flip",
+    "jpeg_compression_distortion",
+    "noise.shot",
+    "reshape",
+    "reinterpret",
+    "water",
+    "sphere",
+    "erase",
+    "random_resized_crop",
+    "ssd_random_crop",
+    "bbox_paste",
+    "coord_flip",
+    "cat",
+    "bb_flip",
+    "warp_affine",
+    "normalize",
+    "pad",
+    "preemphasis_filter",
+    "power_spectrum",
+    "spectrogram",
+    "to_decibels",
+    "sequence_rearrange",
+    "normal_distribution",
+    "mel_filter_bank",
+    "nonsilent_region",
+    "one_hot",
+    "copy",
+    "resize_crop_mirror",
+    "fast_resize_crop_mirror",
+    "segmentation.select_masks",
+    "slice",
+    "segmentation.random_mask_pixel",
+    "transpose",
+    "paste",
+    "mfcc",
+    "lookup_table",
+    "element_extract",
+    "arithmetic_generic_op",
+    "box_encoder",
+]
+
+excluded_methods = [
+    "readers.nemo_asr",
+    "roi_random_crop",
+    "squeeze",
+    "jitter",
+    "permute_batch",
+    "peek_image_shape",
+    "batch_permutation",
+    "expand_dims",
+    "noise.gaussian",
+    "grid_mask",
+    "nemo_asr_reader",
+    "coord_transform",
+    "multi_paste",
+    "segmentation.random_object_bbox",
+    "noise.salt_and_pepper",
+    "dl_tensor_python_function",
+    "hidden.arithmetic_generic_op", #internal
+    "hidden.transform_translation", #internal
+    "video_reader",         # not supported for CPU
+    "video_reader_resize",  # not supported for CPU
+    "readers.video",        # not supported for CPU
+    "readers.video_resize", # not supported for CPU
+    "optical_flow",         # not supported for CPU
+]
+def test_coverage():
```

Comment:
Please put it in some test utils and reuse in both cases.

Reply:
Done